docs/content/preview/yugabyte-voyager/known-issues/postgresql.md

```diff
@@ -1461,4 +1465,128 @@ CREATE TABLE test_jsonb_chk (
     data1 jsonb,
     CHECK (data1->'key'<>'{}')
 );
-```
\ No newline at end of file
+```
+
+### Events Listen / Notify
+
+**GitHub**: Issue [#1872](https://github.com/yugabyte/yugabyte-db/issues/1872)
+
+**Description**: If your application queries or PL/pgSQL objects rely on **LISTEN/NOTIFY events** in the source PostgreSQL database, these functionalities will not work after migrating to YugabyteDB. Currently, LISTEN/NOTIFY events are a no-op in YugabyteDB, and any attempt to use them will trigger a warning instead of performing the expected event-driven operations:
+
+```sql
+WARNING:  LISTEN not supported yet and will be ignored
+```
+
+**Workaround**: Currently, there is no workaround.
+
+**Example:**
+
+```sql
+LISTEN my_table_changes;
+INSERT INTO my_table (name) VALUES ('Charlie');
+NOTIFY my_table_changes, 'New row added with name: Charlie';
+```
+
+### Two-Phase Commit
+
+**GitHub**: Issue [#11084](https://github.com/yugabyte/yugabyte-db/issues/11084)
+
+**Description**: If your application queries or PL/pgSQL objects rely on **Two-Phase Commit** protocol that allows multiple distributed systems to work together in a transactional manner in the [source](https://www.postgresql.org/docs/current/two-phase.html) PostgreSQL database, these functionalities will not work after migrating to YugabyteDB. Currently, Two-Phase Commit is not implemented in the YugabyteDB and will throw the following error when attempt to execute the commands:
+
+```sql
+ERROR:  PREPARE TRANSACTION not supported yet
+```
+
+**Workaround**: Currently, there is no workaround.
+
+## Limitations
```

Comment:
I got this section reviewed by the team and we decided to move this Limitations section to the Schema Review workarounds top [page](https://deploy-preview-25866--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-voyager/known-issues/) and at the end of it with heading `Assess and Analyze Limitations`

Reply:
I've made the changes please look